community/sustainability_effort.md

```diff
@@ -0,0 +1,58 @@
+* [triaging](../docs/CONTRIBUTING.md#triaging-bugs) new incoming bugs by prioritizing them with `P0`, `P1`, `P2`, and `P3` labels
+* responding to questions in [Github Discussions](https://github.com/argoproj/argo-workflows/discussions)
+* responding to questions in [CNCF Slack](https://argoproj.github.io/community/join-slack) in the `#argo-workflows` and `#argo-wf-contributors` channels 
+
+## Commitment
+
+Argo Workflows has roles as described in the [Contributing](../docs/CONTRIBUTING.md#roles) document.
+Any code contributor in a designated role or with an open Membership request can sign up to participate in this Incentive System.
+
+Participants are expected to try to advance in roles.
+There are different expectations depending on the role:
+- Member and non-member: should average a minimum of 6 hours per week of either [authoring PRs](../docs/CONTRIBUTING.md#authoring-prs) or performing any activities listed above in the ["Where is help needed?"](#where-is-help-needed) section
+  - Note that more time is required in this role in order to learn the codebase and processes to make sufficient progress
+- Reviewer and above: should average a minimum of 2 hours per week performing any activities listed above in ["Where is help needed?"](#where-is-help-needed) section 
+  - Highest priority is to ensure that all PRs labeled ["prioritized-review"](#authoring-prs) have an Assignee
+
+Note that the hours per week listed above is an average over time; it's fine to have weeks of no activity so long as the average meets expectations.
+
+Participants should join the [#argo-wf-contributors](https://cloud-native.slack.com/archives/C0510EUH90V) slack channel.
+
+### Reviewing PRs
+
+First priority is to review any PRs which have the `prioritized-review` label, meaning they were authored by a participant of this system (verified by GitHub ID).
+Each of these should have an Assignee: a Reviewer, Approver, or Lead who "owns" reviewing the PR.<br />
+These PRs should be given an initial review within a week.
+The Assignee should then respond to each question or requested modification from the author within a week.
+
+Next, look for PRs with no Assignee. 
+
+For general guidance regarding PR Reviews, see [Reviewing PRs](../CONTRIBUTING.md#reviewing-prs).
+
+### Authoring PRs
+
+Participants can apply a `prioritized-review` label to any PRs they author.
+
+## Participants
+
+If you'd like to participate, add yourself here in a PR.
+
+| Name                      | GitHub ID                                               |
+|---------------------------|---------------------------------------------------------|
+| Julie Vogelman            | [juliev0](https://github.com/juliev0)                   |
+| Saravanan Balasubramanian | [sarabala1979](https://github.com/sarabala1979)         |
+| Yuan Tang                 | [terrytangyuan](https://github.com/terrytangyuan)       |
+| Alan Clucas               | [Joibel](https://github.com/Joibel)                     |
```

Comment:
Can add me now as I definitely meet (or exceed) the current expectations
```suggestion
| Alan Clucas               | [Joibel](https://github.com/Joibel)                     |
| Anton Gilgur              | [agilgur5](https://github.com/agilgur5)                |
```

Also we probably want to alphabetize these? Right now they are assorted?
And we may want to put GitHub IDs in backticks

Reply:
Thanks for joining the effort! I know of course you already put in much more than the required minimum hours.

Reply:
It's interesting that the lack of backticks doesn't trigger the spelling error. Perhaps there's an exception for links?

Reply:
Yea I noticed that too, I think `mdspell` may not be looking at the `community/` directory at all and [only the `docs/` directory right now](https://github.com/argoproj/argo-workflows/blob/c71e1dc05a052778bed597f7c0f50b2ede4a7f32/Makefile#L687)

Reply:
I see